packages/zoe/src/instanceRecordStorage.js

```diff
@@ -60,26 +59,49 @@ export const makeInstanceRecordStorage = instanceRecord => {
     };
   };
 
-  const getInstanceRecord = () => harden(instanceRecord);
-  const getTerms = () => instanceRecord.terms;
-  const getIssuers = () => instanceRecord.terms.issuers;
-  const getBrands = () => instanceRecord.terms.brands;
+  /** @type {ExportInstanceRecord} */
+  const exportInstanceRecord = () => {
+    assertInstantiated();
+    return harden(instanceRecord);
+  };
+  const getTerms = () => {
+    assertInstantiated();
+    return instanceRecord.terms;
+  };
+  const getIssuers = () => {
+    assertInstantiated();
+    return instanceRecord.terms.issuers;
+  };
+  const getBrands = () => {
+    assertInstantiated();
+    return instanceRecord.terms.brands;
+  };
 
   const assertUniqueKeyword = keyword => {
+    assertInstantiated();
     assertKeywordName(keyword);
     assert(
       !getKeywords(instanceRecord.terms.issuers).includes(keyword),
       X`keyword ${q(keyword)} must be unique`,
     );
   };
 
+  const instantiate = startingInstanceRecord => {
+    assert(
+      instanceRecord === undefined,
+      X`instanceRecord ${instanceRecord} can only be instantiated once`,
+    );
+    instanceRecord = startingInstanceRecord;
+  };
+
   return harden({
     addIssuerToInstanceRecord,
-    getInstanceRecord,
+    exportInstanceRecord,
```

Comment:
Why `export`? I see there were several uses before this PR but I'm still curious.

Reply:
Why `export` rather than `get`? I wanted the API to have the same format for both the `issuerRecordStorage` and the `instanceRecordStorage`. In both storage cases, the methods are called specifically to export the data to ZCF in the contract instance vat. In the case of the issuerRecord, a new format is used for exporting, so `get` doesn't seem fully appropriate.

Reply:
I'm worried that we already use two distinct meanings of `export`/`import`: between JS modules, and over the liveSlots/capTP c-lists. Not a fatal objection, but it would be good to find a non-overlapping word.